node/src/chain/acceptor.rs

```diff
@@ -904,6 +907,22 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
                 };
 
                 vm.finalize_state(new_final_state_root, old_final_state_roots)?;
+
+                /// The initial finalization period in blocks, equivalent to at
+                /// least 7 days: max 6 blocks per min * 60 * 24 * 7
+                pub const INITIAL_FINALIZATION_PERIOD: u64 = 60_480u64;
```

Comment:
We need to move this somewhere else

Reply:
The finalization period is a concept related to the bridge. Here we should have a "challenge" period, although we don't really have challenges...

Reply:
I'll put this in conf with a default value

Hence I commented, just to remind myself about it
Do you have any suggestion about name and default value?